internal/services/healthcare/healthcare_dicom_resource_test.go

```diff
@@ -119,78 +105,109 @@ func (r HealthCareDicomResource) basic(data acceptance.TestData) string {
 %s
 
 resource "azurerm_healthcare_dicom_service" "test" {
-  name         = "dicom%d"
+  name         = "dicom%[2]s"
   workspace_id = azurerm_healthcare_workspace.test.id
-  location     = "%s"
+  location     = azurerm_resource_group.test.location
   depends_on   = [azurerm_healthcare_workspace.test]
+
+  lifecycle {
+    ignore_changes = [
+      "cors",
+      "storage"
+    ]
+  }
```

Comment:
Why do we need this? Does the API have defaults for these?

Reply:
yes, api is returning default values for these causing a plan diff when it is not declared in the terraform script

Reply:
@hqhqhqhqhqhqhqhqhqhqhq in that case we probably want to mark these two blocks as `Computed`, otherwise users that upgrade and don't want to set this block will be forced to either add them to their config or specify them in `ignore_changes` which would be a breaking change.

When you mark the blocks as `Computed` can you please add a comment above it with the following format:
`// NOTE: O+C API sets defaults for these`, it's so that we have a consistent identifier for `Optional` + `Computed` properties throughout the provider.